pkg/flag/rego_flags.go

```diff
@@ -7,6 +7,11 @@ package flag
 //	  config-policy: "custom-policy/policy"
 //	  policy-namespaces: "user"
 var (
+	IncludeDeprecatedChecksFlag = Flag[bool]{
```

Comment:
The [config-file.md](https://github.com/aquasecurity/trivy/blob/main/docs/docs/references/configuration/config-file.md) should be updated.

Reply:
Thanks, updated.